### ISSUE TITLE
Cannot Disable GraphQL ESLint Rules in Typescript

### ISSUE THREAD

**Issue by matthew-valenti** [dotansimha/graphql-eslint#622]
**Describe the bug**

**To Reproduce**
In Typescript, ESLint shows "parsing error" for this line even though it's marked as ESLint disabled.
```
// eslint-disable-next-line
const test = gql`test`;
```

**Expected behavior**
In Typescript, Do not lint code if marked with standard ESLint disabled flags.

**Environment:**

- OS: Windows 10 / VSCode
- `@graphql-eslint/eslint-plugin": "^2.1.0"`
- eslint: 7.32.0
- NodeJS: v14.16.1
- typescript: 4.x

**Additional context**
The documentation shows this is an example:
```# eslint-disable-next-line```
However, this is not valid comment in typescript.
I've tried every ESLint ignore rule e.g. ```/* eslint-disable */```

**Temporary Workaround**
Add .eslintignore to project root and ignore whole file.

**Comment by B2o5T** [comment]
Hi @matthew-valenti, firstly your document contains invalid syntax. String `test` means nothing in GraphQL, try `{ test }` that means query `test` resolver, in this case, you won't get parsing error.

Secondly, to disable GraphQL rules on code files (JavaScript or TypeScript) you must put `eslint-disable` comment inside backticks content. Like this:

```js
const test = gql`
  # eslint-disable-next-line
  query {
    test
  }
`;
```
